src/couch/src/couch_bt_engine.hrl

```diff
@@ -14,8 +14,7 @@
     filepath,
     fd,
     fd_monitor,
-    % deprecated but keeping it here to avoid altering the record size
-    fsync_options_deprecated,
+    cache = #{},
```

Comment:
`fsync_options_deprecated` is a deprecated field and I haven't found any other
source code related to it, so I guess replacing it with `cache` should be fine.